### ISSUE TITLE
pg_stat_statements scan failed: sql: Scan error on column index 3, name "left": converting NULL to string is unsupported

### ISSUE THREAD

**Issue by Nipsuli** [rapidloop/pgmetrics#29]
Getting following error:
```
pgmetrics: pgmetrics scan failed: sql: Scan error on column index 3, name "left": converting NULL to string is unsupported
```
from time to time. Running `select pg_stat_statements_reset` helps, but it does come back
pgmetrics version: `1.8.1`
postgres version: `12.1 (Ubuntu 12.1-1.pgdg18.04+1)`

**Comment by Nipsuli** [comment]
I guess this is actually postgres "feature"
> The representative query texts are kept in an external disk file, and do not consume shared memory. Therefore, even very lengthy query texts can be stored successfully. However, if many long query texts are accumulated, the external file might grow unmanageably large. As a recovery method if that happens, pg_stat_statements may choose to discard the query texts, whereupon all existing entries in the pg_stat_statements view will show null query fields, though the statistics associated with each queryid are preserved. If this happens, consider reducing pg_stat_statements.max to prevent recurrences.

https://www.postgresql.org/docs/12/pgstatstatements.html

but still, I'd like `pgmetrics` to not fail totally, but give some sort of error in the output, so other metrics would be shipped.

**Comment by mdevan** [comment]
Fixed with c66edb8, thanks for reporting.
